zerver/test_external.py

```diff
@@ -224,4 +264,3 @@ def test_change_user(self):
         self.login("hamlet@zulip.com")
         result = self.client.post('/json/users/me/android_gcm_reg_id', {'token':token})
         self.assert_json_success(result)
-
```

Comment:
For some reason my text editor keeps removing that line whenever I try to save the file. I'll make sure not to add it to a commit next time